components/console_cmd_mqtt/README.md

```diff
@@ -0,0 +1,56 @@
+# Console command mqtt
+The component provides a console where the 'mqtt' command can be executed.
+
+## API
+
+### Steps to enable console in an example code:
```

Comment:
I'm missing explanation on how to do configure the mqtt client for different options.